### DISCUSSION TITLE
Would you guys be willing to make a new example tutorial with VueFlow that implements a TipTap rich text editor?

### DISCUSSION THREAD

**Discussion by SkyAphid** [bcakmakoglu/vue-flow#1652] [Ideas]
Hello again! 

So, my project uses VueFlow as I've mentioned in other issues I've reported, but it also uses VueQuill as well. The goal of my program is to basically be a node-based text editor, so naturally a rich text editor is required. You can check out the project here, and see for yourself how the implementation is being used:

https://skyaphid.github.io/corkboard/

The problem is that VueQuill is heavily outdated and has some ongoing problems that aren't being addressed by developers. There's an ongoing issue right now that has been open for months now where people including myself have discussed its status as basically being a deprecated piece of software: https://github.com/vueup/vue-quill/issues/589#issuecomment-2415700842

So there's basically a mass exodus to TipTap going on right now. I've personally tried to migrate to  TipTap myself, but have had some issues getting the Nodes, NodeResizer, and TipTap to all place nicely with each other, whereas VueQuill mostly "just worked" when I plugged it in. 

I was wondering if you guys could possibly look into making an example on VueFlow's website where Node, NodeResizer, and TipTap are all being used together in a fully functioning way. There's no rush either, because as of right now VueQuill is performing it's job just fine- but I would likely to eventually move to it.

Thank you for your time!

**Comment by bcakmakoglu** [comment]
“You guys” - it’s quite literally just me maintaining the library 😅
I’m not sure if I have the time in the near future to do something like that and whether it’s even worth having an example that sounds like it’s made for a very specific use-case.
The examples are meant to just give you an idea how to handle general concepts and not how to solve more specific use-cases.

If you’re willing to do an example im always open to merging it into the docs.
Or just ask some questions about issues you’re facing in the implementation and I’ll try my best to answer them (although I’m on vacation atm so it’ll be a while until I can properly respond).

**Comment by SkyAphid** [comment]
Alright, at the very least in the future I can probably just make a small codebox example of how this and even the most basic Tiptap example aren't wanting to work well together and get some feedback in that case. Also, my mistake! I thought I saw multiple people chiming in on the other issue I posted.
